lib/iris/tests/integration/test_pp.py

```diff
@@ -587,5 +588,52 @@ def test_as_pairs(self):
             self.assertEqual(cube.attributes['pseudo level'], 3)
 
 
+class TestSaveLBPROC(tests.IrisTest):
+
+    def create_cube(self):
+        cube = Cube(np.zeros((2, 3, 4)))
+        tunit = Unit('days since 0001-01-01 00:00:00', calendar='gregorian')
+        tcoord = DimCoord(np.arange(2), standard_name='time', units=tunit)
+        xcoord = DimCoord(np.arange(3), standard_name='longitude',
+                          units='degrees')
+        ycoord = DimCoord(points=np.arange(4))
+        cube.add_dim_coord(tcoord, 0)
+        cube.add_dim_coord(xcoord, 1)
+        cube.add_dim_coord(ycoord, 2)
+        return cube
+
+    def convert_cube_to_field(self, cube):
+        field = iris.fileformats.pp.PPField3()
+        field.lbvc = 0
+        iris.fileformats.pp._ensure_save_rules_loaded()
+        iris.fileformats.pp._save_rules.verify(cube, field)
+        return field
+
+    def test_time_mean_only(self):
+        cube = self.create_cube()
+        cube.cell_methods = (
+            iris.coords.CellMethod(method='mean', coords='time'),
+        )
```

Comment:
@hdyson 
A minor point, but it's probably safer (from a maintenance point of view) to use the `add_cell_method` method ...
```python
cm = CellMethod(method='mean', coords='time')
cube.add_cell_method(cm)
```

Note that, `CellMethod` is imported directly `from iris.coords` ... so there's no need to specify the full namespace i.e. `iris.coords.CellMethod`

If you agree, then rinse-and-repeat for all below ... ?